tests/unit/HelperTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Mockery as m;
+
+class HelperTest extends PHPUnit_Framework_TestCase
+{
+	/** @test */
+	public function helper_without_parameters_returns_store() 
+	{
+		$this->assertInstanceOf('anlutro\LaravelSettings\SettingStore', setting());
+	}
+
+	/** @test */
+	public function single_parameter_get_a_key_from_store() 
+	{
+		app()->shouldReceive('get')->with('foo', null)->once();
+
+		setting('foo');
+	}
+
+
+	/** @test */
+	public function two_parameters_set_to_store() 
+	{
+		app()->shouldReceive('set')->with('foo', 'bar')->once();
+
+		setting('foo', 'bar');
+	}
+}
+
+$containerInstance = m::mock('anlutro\LaravelSettings\SettingStore');
+
+function app() {
```

Comment:
Causing issues with test redeclaring as it is already declared in /vendor/illuminate/support/Illuminate/Support/helpers.php:26

Reply:
Will dig into that, I have a few more improves as well

Reply:
Happens when testing with Laravel 4.

Reply:
Thanks, I have some more spare time tomorrow, will try to fix

Reply:
Cool, would be a great addition 👍